packages/extremetuxracer.rb

```diff
@@ -0,0 +1,34 @@
+require 'package'
+
+class Extremetuxracer < Package
+  description 'High speed arctic racing game based on Tux Racer'
+  homepage 'https://sourceforge.net/projects/extremetuxracer/'
+  version "0.8.4-1"
+  compatibility 'x86_64 armv7l aarch64'
+  license 'GPLv2'
+  source_url ({
+    "armv7l": "https://github.com/FinnBaltazar1111/extremetuxracer/releases/download/0.8.4/extremetuxracer_#{version}_armhf.deb",
+    "x86_64": "https://github.com/FinnBaltazar1111/extremetuxracer/releases/download/0.8.4/extremetuxracer_#{version}_amd64.deb",
+    "aarch64": "https://github.com/FinnBaltazar1111/extremetuxracer/releases/download/0.8.4/extremetuxracer_#{version}_armhf.deb"
```

Comment:
This looks correct, although we sort the architectures in all our other packages.